WeScanTests/UIImageTests.swift

```diff
@@ -65,7 +65,7 @@ final class UIImageTests: FBSnapshotTestCase {
         let view = UIImageView(image: orientatedImage.applyingPortraitOrientation())
         view.sizeToFit()
         
-        FBSnapshotVerifyView(view)
+        FBSnapshotVerifyView(view, perPixelTolerance: 6/256)
```

Comment:
<!--
  0 failure: 
  1 warning:  Operators should ...
  
  
  DangerID: danger-id-Danger;
  File: WeScanTests/UIImageTests.swift;
  Line: 68;
-->

- :warning: Operators should be surrounded by a single whitespace when they are being used. (`operator_usage_whitespace`)